app.js

```diff
@@ -1,11 +1,61 @@
-var express = require('express');
-var app = express();
+// Load libraries
+var express = require('express')
+  , app = express()
+  , mongoose = require('mongoose')
+  , bodyParser = require('body-parser')
+  , morgan = require('morgan')
+  , cookieParser = require('cookie-parser')
+  , expressSession = require('express-session')
+  , csrf = require('csurf')
+  , handlebars  = require('express-handlebars');
 
-app.get('/', function(req, res) {
-  res.send('Hello, world!');
+// Load configuration and routes
+var port = process.env.PORT || 3000 // port 3000 as default
+  , passport = require('./config/passport')
+  , DB = require('./config/database')
+  , authRoutes = require('./routes/auth');
```

Comment:
I think it'd be cleaner to initialize these variables just before they're used. I.e. the `port` declaration just before `app.listen`, `passport` just before `app.use(passport.initialize());`, `DB` just before `mongoose.connect(DB.url)`, and `authRoutes` just before its `app.use` statement.